AuthenticatorShared/UI/Platform/Application/Support/Localizations/en.lproj/Localizable.strings

```diff
@@ -122,3 +122,8 @@
 "Backup" = "Backup";
 "BitwardenAuthenticatorDataIsBackedUpAndCanBeRestored" = "Bitwarden Authenticator data is backed up and can be restored with your regularly scheduled device backups.";
 "ImportingFromTwoFasPasswordProtectedNotSupported" = "Importing from 2FAS password protected files is not supported. Try again with an exported file that is not password protected.";
+"WithTheBitwardenAppYouCanStore" = "With the Bitwarden app, you can store all of your items and sync directly with the authenticator.";
+"DownloadTheBitwardenApp" = "Download the Bitwarden app";
+"InOrderToViewAllOfYourVerificationCodes" = "In order to view all of your verification codes, you’ll need to allow for syncing on all of your accounts.";
```

Comment:
⛏️ In the password manager app we have keys that have around 90 characters in length so you could have them aligned here too. Additionally, when they are not aligned in the password manager app we're using a suffix `DescriptionLong` to be added to the key to denote that the value is longer than what the key has.